src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/PersistorBase.java

```diff
@@ -120,9 +120,7 @@ abstract void createTable(
   public synchronized String read() {
     // Currently this method only contains a call to readTables() - later could add the part to read
     // multiple sqlite files
-    LOG.debug("RCA: in read() in PersistorBase");
```

Comment:
I think we should add a class level javadoc comment saying what are the tables in the SQLite database and what are the columns for each. And also add an example tables filled with data for each

Reply:
done. Added to class level javadoc of each summary class.